Makefile

```diff
@@ -1,10 +1,16 @@
 #
 # Makefile - `gocat` (https://github.com/tuladhar/gocat)
 # 
-PROGRAM_NAME:=gocat
+PROGRAM_NAME=gocat
+INSTALL_PATH=/usr/local/bin
 
-build:
```

Comment:
Let's have `install` target separately rather than mixing both build and install.